### ISSUE TITLE
Deployment with Docket doesn't work

### ISSUE THREAD

**Issue by ChenMorSays** [lugia19/elevenlabslib#18]
This lib uses sounddevice and it wouldn't work on the cloud FYI

**Comment by lugia19** [comment]
Yeah, this was a conscious choice - I wanted maximum flexibility locally, so I figured it wouldn't work in docker/etc.

**Comment by IveDeletedYourPostAsItsADuplicate** [comment]
Just wanted to correct this comment. Deployment with Docker does work (I've been running it in docker for the past couple of months with great results!)

Generating audio without playback features works fine. It's only when playback on device is used that this issue appears.

**Comment by lugia19** [comment]
Interesting - do you have any more details on how it doesn't work when trying to select a playback device? Maybe an error log?

Also, try doing `sounddevice.query_devices()`. It should give you a list of every device, with corresponding IDs.
